controllers/domachine_controller.go

```diff
@@ -259,6 +288,32 @@ func (r *DOMachineReconciler) reconcile(ctx context.Context, machineScope *scope
 		return reconcile.Result{}, nil
 	}
 }
+func (r *DOMachineReconciler) reconcileDeleteVolumes(ctx context.Context, mscope *scope.MachineScope, cscope *scope.ClusterScope) (reconcile.Result, error) {
+	mscope.Info("Reconciling delete DOMachine Volumes")
+	computesvc := computes.NewService(ctx, cscope)
+	domachine := mscope.DOMachine
+	for _, disk := range mscope.DOMachine.Spec.DataDisks {
```

Comment:
Does it make sense to use the `domachine` variable?

Something like:
```
for _, disk := range domachine.Spec.DataDisks {
```

Reply:
done